contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+
+This article is US-focused, but most of it applies to any country. I won’t offer multiple options here - these are the tools that I have settled on using personally after a bunch of research and trying out different things.
+
+By using these, hopefully I can save you a bunch of time and precious $$$ trying to figure out how to put together contracts, pay people on time, manage your money and comply with laws you are only just finding out exist.
+
+## Paying people
+
+You’re going to be hiring people either as permanent employees or as contractors. You will need the appropriate contract, and a system for automatically paying them.
+
+<iframe src="https://giphy.com/embed/l0G18S6f5RfkI8JNK" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/season-12-the-simpsons-12x14-l0G18S6f5RfkI8JNK">via GIPHY</a></p>
+
+### Permanent employees
+
+I like <a target="_blank" rel="noopener noreferrer nofollow" href="https://gusto.com/">Gusto</a> a lot (US only). They make the process pretty seamless for you and your new hire, offer tantalizing discounts for early stage startups, and integrate things like state tax registrations, healthcare benefits and mandatory insurance quite nicely. Their phone support is great, but avoid email or chat. Gusto charge $39/month plus $6 per employee.
+
+Time suck alert - online state registration laws vary and all of their websites are garbage. Gusto have a ton of useful documentation on this, but bear in mind you will probably spend 3 hours registering in each state the first time you employ someone there. <a target="_blank" rel="noopener noreferrer nofollow" href="https://agent.middesk.com/">Middesk</a> can handle this for you for $99, but I haven’t used it (yet).
+
+### Contractors
+
+<a target="_blank" rel="noopener noreferrer nofollow" href="https://www.letsdeel.com/">Deel</a> is the only way to go. They cover pretty much any country and currency you’ll need, handle all the compliance stuff for you, and generally make the process way more pleasant than it has any right to be. Plus their support is insanely responsive and helpful. It’s probably my favourite ops product. Deel charge $35/month per contractor.
+
+If you need an Employer of Record service, Deel now offer this too - super useful if you are hiring permanent employees outside the US in a country where you don’t have a legal presence.
+
+### Contracts
+
+Orrick have a <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.orrick.com/en/Total-Access/Tool-Kit/Start-Up-Forms/Employment-and-Consultant">handy free set of templates</a> you can use! (US only)
+
+You need both an Offer Letter and a Confidential Information and Invention Assignment Agreement. Deel have free templates integrated into their platform already, so you’ll only usually use these for permanent employees, unless you want to use a customized contractor agreement in Deel.
```

Comment:
maybe just add a very brief explanation about why these matter

Reply:
"The CIIAA means that your employees agree that their work is confidential and whatever they build at work is owned by the company. Not having this = major problems in future, especially for fundraising."